packages/app/src/cli/models/app/app.ts

```diff
@@ -127,38 +60,27 @@ export const NonVersionedAppBottomSchema = zod.object({
 })
 export const NonVersionedAppSchema = NonVersionedAppTopSchema.merge(NonVersionedAppBottomSchema)
 
-export const VersionedAppSchema = zod.object({
-  application_url: validateUrl(zod.string()),
-  embedded: zod.boolean(),
-  access: zod
-    .object({
-      direct_api_offline_access: zod.boolean().optional(),
-    })
-    .optional(),
-  webhooks: WebhooksSchemaWithDeclarative,
-  app_proxy: zod
-    .object({
-      url: validateUrl(zod.string()),
-      subpath: zod.string(),
-      prefix: zod.string(),
-    })
-    .optional(),
-  pos: zod
-    .object({
-      embedded: zod.boolean(),
-    })
-    .optional(),
-  app_preferences: zod
-    .object({
-      url: validateUrl(zod.string().max(255)),
-    })
-    .optional(),
-})
-
-export const AppSchema = NonVersionedAppTopSchema.merge(VersionedAppSchema).merge(NonVersionedAppBottomSchema).strict()
+export const AppSchema = NonVersionedAppTopSchema.merge(NonVersionedAppBottomSchema).strict()
```

Comment:
This is invalid now no?

Reply:
you are right!, once we removed the `VersionedAppSchema` one the `NonVersionedAppSchema` is not needed anymore.
I will remove the `NonVersioned` schemas definitively in one of the follow up PRs once the `app_access` and `branding` modules are integrated